lib/network/options.js

```diff
@@ -667,6 +669,8 @@ let configureOptions = {
     selectConnectedEdges: true,
     hoverConnectedEdges: true,
     tooltipDelay: [300, 0, 1000, 25],
+    zoomMin: 0.00001,
+    zoomMax: 10,
```

Comment:
This should be a range of possible values (like the value right above it). The configurator isn't able to handle single number.

Reply:
Updated.